src/azure-cli/azure/cli/command_modules/acs/custom.py

```diff
@@ -779,6 +781,9 @@ def aks_update(
     yes=False,
     no_wait=False,
     aks_custom_headers=None,
+    # metrics profile
+    enable_cost_analysis=False,
+    disable_cost_analysis=False,
```

Comment:
These two default values are both False, which seems contradictory. Could they both be None?

Reply:
It's not really contradictory. If you are neither enabling or disabling this feature, then the default behavior is to leave it alone (its value stays whatever it was before).

There are many other pairs of enable/disables with the same semantics in the AKS CLI, see for example:
```
    enable_image_cleaner=False,
    disable_image_cleaner=False,
    enable_keda=False,
    disable_keda=False,
    enable_vpa=False,
    disable_vpa=False,
    enable_disk_driver=False,
    disable_disk_driver=False,
    enable_file_driver=False,
    disable_file_driver=False,
    enable_workload_identity=False,
    disable_workload_identity=False,
    enable_snapshot_controller=False,
    disable_snapshot_controller=False,
    enable_azure_keyvault_kms=False,
    disable_azure_keyvault_kms=False,
```

(there are more).

Throwing `None` in the mix doesn't really clarify things, as `None` behaves just like `False` would and so there becomes no difference between `enable=false, disable=false` and `enable=None, disable=None`.

Reply:
As a supplement, aks command group usually provides a pair of options for setting on or off for a feature (`--enable-xxx`/`--disable-xxx`). Based on the previous discussion, to maintain the consistency of similar options, AKS PM decided not to use the three-state flag, so `False` here does not really mean `False` (~~`enable=False` -> `disable`~~), but means `None`.

Reply:
Got it, thanks~